### DISCUSSION TITLE
What is the functionality of OrderMutation

### DISCUSSION THREAD

**Discussion by mxmxlwlw** [apache/incubator-mxnet#8529] [Q&A]
Hi,
    I noticed that the ordermutation pass of graph doesn't be used. I guess that's maybe because of the dependency engine. I want to know how to achieve the writing and reading order by using ordermutation. Thanks a lot.

**Comment by mxmxlwlw** [comment]
And I want to give a suggestion. Please don't do the pass in just one function. Make it a class, and implement it by using more than one function. Or just give us more docs about it.

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by rahul003** [comment]
@eric-haibin-lin please add the labels Doc, Question
